keyboards/rmkeebs/rm_fullsize/info.json

```
@@ -0,0 +1,155 @@
+    "keyboard_name": "rm_fullsize",
+    "maintainer": "RuckerMachine",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["GP22", "GP23", "GP24", "GP25", "GP26", "GP27", "GP28", "GP29", "GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP12", "GP13", "GP14", "GP16", "GP17"],
+        "rows": ["GP8", "GP7", "GP9", "GP20", "GP18", "GP19"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "GP11", "pin_b": "GP10" }
+        ]
+    },
+    "layouts": {
+        "LAYOUT_all_split": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0.25 },
+                { "matrix": [0, 2], "x": 2, "y": 0.25 },
+                { "matrix": [0, 3], "x": 3, "y": 0.25 },
+                { "matrix": [0, 4], "x": 4, "y": 0.25 },
+                { "matrix": [0, 5], "x": 5, "y": 0.25 },
+                { "matrix": [0, 6], "x": 6.5, "y": 0.25 },
+                { "matrix": [0, 7], "x": 7.5, "y": 0.25 },
+                { "matrix": [0, 8], "x": 8.5, "y": 0.25 },
+                { "matrix": [0, 9], "x": 9.5, "y": 0.25 },
+                { "matrix": [0, 10], "x": 11, "y": 0.25 },
+                { "matrix": [0, 11], "x": 12, "y": 0.25 },
+                { "matrix": [0, 12], "x": 13, "y": 0.25 },
+                { "matrix": [0, 13], "x": 14, "y": 0.25 },
+                { "matrix": [0, 14], "x": 15.25, "y": 0.25 },
+                { "matrix": [0, 15], "x": 16.25, "y": 0.25 },
+                { "matrix": [3, 15], "x": 17.25, "y": 0.25 },
+                { "matrix": [0, 16], "x": 18.5, "y": 0.25 },
+                { "matrix": [0, 17], "x": 19.5, "y": 0.25 },
+                { "matrix": [0, 18], "x": 20.5, "y": 0.25 },
+                { "matrix": [0, 19], "x": 21.75, "y": 0 },
+
+                { "matrix": [1, 0], "x": 0, "y": 1.5 },
+                { "matrix": [1, 1], "x": 1, "y": 1.5 },
+                { "matrix": [1, 2], "x": 2, "y": 1.5 },
+                { "matrix": [1, 3], "x": 3, "y": 1.5 },
+                { "matrix": [1, 4], "x": 4, "y": 1.5 },
+                { "matrix": [1, 5], "x": 5, "y": 1.5 },
+                { "matrix": [1, 6], "x": 6, "y": 1.5 },
+                { "matrix": [1, 7], "x": 7, "y": 1.5 },
+                { "matrix": [1, 8], "x": 8, "y": 1.5 },
+                { "matrix": [1, 9], "x": 9, "y": 1.5 },
+                { "matrix": [1, 10], "x": 10, "y": 1.5 },
+                { "matrix": [1, 11], "x": 11, "y": 1.5 },
+                { "matrix": [1, 12], "x": 12, "y": 1.5 },
+                { "matrix": [1, 13], "w": 1, "x": 13, "y": 1.5 },
```

```suggestion
                { "matrix": [1, 13], "x": 13, "y": 1.5 },
```
Default width and height is 1u. Please remove these for the other keys.